pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -101,13 +84,118 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	// Allow traffic into the API (port 443) from KubernetesAPIAccess CIDRs
 	{
+		network, err := b.LinkToNetwork()
+		if err != nil {
+			return err
+		}
 		b.AddFirewallRulesTasks(c, "https-api", &gcetasks.FirewallRule{
 			Lifecycle:    b.Lifecycle,
-			Network:      b.LinkToNetwork(),
+			Network:      network,
 			SourceRanges: b.Cluster.Spec.KubernetesAPIAccess,
 			TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleMaster)},
 			Allowed:      []string{"tcp:443"},
 		})
 	}
 	return nil
+
+}
+
+// createInternalLB creates an internal load balancer for the cluster.  In
+// GCP this entails creating a health check, backend service, and one forwarding rule
+// per specified subnet pointing to that backend service.
+func createInternalLB(b *APILoadBalancerBuilder, c *fi.ModelBuilderContext) error {
+	lbSpec := b.Cluster.Spec.API.LoadBalancer
+	hc := &gcetasks.HealthCheck{
+		Name:      s(b.NameForHealthCheck("api")),
+		Port:      443,
+		Lifecycle: b.Lifecycle,
+	}
+	c.AddTask(hc)
+	var igms []*gcetasks.InstanceGroupManager
+	for _, ig := range b.InstanceGroups {
+		if ig.Spec.Role != "Master" {
```

Comment:
Nit: It's a good idea to use the constants (InstanceGroupRoleMaster), particularly where at some stage we'll want to address the legacy naming here.

Reply:
Done.